pylint/checkers/similar.py

```diff
@@ -235,7 +235,7 @@ def hash_lineset(
     index2lines = {}
     # Comments, docstring and other specific patterns maybe excluded -> call to stripped_lines
     # to get only what is desired
-    lines = tuple(x.text for x in lineset.stripped_lines)
+    lines: tuple[str, ...] = tuple(x.text for x in lineset.stripped_lines)
```

Comment:
Let's separate the fix on line too long and the typing, please

Reply:
Ok, I just thought, as long as I was fixing the code, I could try to add some types.

Reply:
It's better to create multiple merge requests, faster to review for us, so we can merge subpart fast and focus on typing only later.